src/pages/workspaces/workspace/Workflows.js

```diff
@@ -205,10 +208,7 @@ export const FindWorkflowModal = ({ namespace, name, ws, onDismiss }) => {
   // Lifecycle
   useOnMount(() => {
     const load = async () => {
-      const [featuredList, methods] = await Promise.all([
-        Ajax(signal).FirecloudBucket.getFeaturedMethods(),
-        Ajax(signal).Methods.list({ namespace: 'gatk' }),
-      ]);
+      const [featuredList, methods] = await Promise.all([FirecloudBucket(signal).getFeaturedMethods(), Methods(signal).list({ namespace: 'gatk' })]);
```

Comment:
Just a heads up, the `FindWorkflowModal` is getting replaced in PR #5184. So these Ajax calls in the modal would no longer be needed.